sonar-csharp-plugin/pom.xml

```diff
@@ -252,6 +252,9 @@
                         <include name="Google.Protobuf.dll"/>
                         <include name="Newtonsoft.Json.dll"/>
                       </fileset>
+                      <fileset dir="${project.build.directory}/../..">
+                        <include name="THIRD-PARTY-NOTICES.txt"/>
```

Comment:
File is missing on my machine in resulting jar file.

Reply:
you can download the JAR for this PR from [burgr](https://burgr.sonarsource.com/projects/SonarSource/sonar-dotnet/pullrequests/4662) - and direct link [on repox](https://repox.jfrog.io/repox/sonarsource/org/sonarsource/dotnet/sonar-csharp-plugin/8.27.0.34588/sonar-csharp-plugin-8.27.0.34588.jar)

I am embedding the license in the ZIP in `static\SonarAnalyzer-8.27.0.34588.zip` - it's together with the dotnet assemblies 

I will investigate how the maven licenses are generated, however I believe that it's good enough to embed them like this for now.